src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -164,7 +165,15 @@ public void Start()
                 });
 
                 server.InboundNodeConnectionParameters = cloneParameters;
-                server.Listen();
+                try
+                {
+                    server.Listen();
+                }
+                catch (SocketException e)
+                {
+                    logger.LogCritical("P2P port is used (is a StratisX QT running?) please use a different port using the '-port config' argument for runing the node.");
```

Comment:
It can also be a bitcoin node that's running and causing trouble. Does the exception give us any interesting data we can include in the log?
Also, `runing` should be `running` and `please...` should be preceded by a full stop and start with an uppercase. 😸

Reply:
>Does the exception give us any interesting data we can include in the log?
Not really.
So I guess optimal solution would be to log the port that can't be listened. 

Sorry about grammar, silly mistakes, should have checked.